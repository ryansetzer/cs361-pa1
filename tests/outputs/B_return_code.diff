--- outputs/B_return_code.txt	2022-09-25 14:55:17.786668184 -0400
+++ expected/B_return_code-4.txt	2022-09-11 14:59:21.000000000 -0400
@@ -0,0 +1,10 @@
+pwd.txt
+empty.txt
+0
+1
+$ ./bin/ls data
+$ echo $?
+$ ./bin/ls asldfkjasldfkj
+$ echo $?
+$ quit
+
