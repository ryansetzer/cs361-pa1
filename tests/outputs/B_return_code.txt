pwd.txt
empty.txt
$ ./bin/ls data
$ echo $?
0
$ ./bin/ls asldfkjasldfkj
$ echo $?
1
$ quit

