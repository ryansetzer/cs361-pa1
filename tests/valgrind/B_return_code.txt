==59392== Memcheck, a memory error detector
==59392== Copyright (C) 2002-2017, and GNU GPL'd, by Julian Seward et al.
==59392== Using Valgrind-3.15.0 and LibVEX; rerun with -h for copyright info
==59392== Command: ../dukesh -b scripts/rc.txt
==59392== 
==59392== 
==59392== HEAP SUMMARY:
==59392==     in use at exit: 0 bytes in 0 blocks
==59392==   total heap usage: 0 allocs, 0 frees, 0 bytes allocated
==59392== 
==59392== All heap blocks were freed -- no leaks are possible
==59392== 
==59392== For lists of detected and suppressed errors, rerun with: -s
==59392== ERROR SUMMARY: 0 errors from 0 contexts (suppressed: 0 from 0)
