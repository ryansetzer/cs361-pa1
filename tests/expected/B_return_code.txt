$ ./bin/ls data
empty.txt
pwd.txt
$ echo $?
0
$ ./bin/ls asldfkjasldfkj
$ echo $?
1
$ quit

